app/src/main/AndroidManifest.xml

```diff
@@ -121,6 +121,12 @@
             android:exported="true"
             android:label="@string/activity_dsl_styling">
         </activity>
+        <activity
+            android:name=".examples.java.RuntimeStylingJavaActivity"
+            android:description="@string/description_dsl_styling"
+            android:exported="true"
+            android:label="@string/activity_dsl_styling">
```

Comment:
Using this strings will actually make 2 duplicate entries in activity list which is no good.
Guess worth adding new ones for label and description stating that Java is used for this example

Reply:
Activity list will only show those with `meta-data`, so this one will be shown in the list. Only be available if you set it as lanch activity in Android studio.

Reply:
perhaps I did not fully get it.
you're saying that this activity will not be shown in `ExampleOverviewActivity` list, correct?

Reply:
That's right